api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder

```diff
@@ -8,6 +8,8 @@ json.is_star_ready app.star_ready?
 json.is_connec_ready app.connec_ready?
 json.is_coming_soon app.coming_soon?
 json.single_billing app.single_billing?
+json.multi_instantiable app.multi_instantiable
+json.subcategories app.subcategories
```

Comment:
yes we do. See: https://github.com/x4d3/mno-enterprise-angular/blob/e44298f3ae80534941c8c0a11a80dbf68fe57932/src/app/views/marketplace/marketplace-app.controller.coffee#L75